py/dml/types.py

```diff
@@ -1271,6 +1296,13 @@ def compatible_types(type1, type2):
         ('size_t', TSize(False))]:
     typedefs[name] = typ
 
+if dml.globals.dml_version != (1, 2):
```

Comment:
You shouldn't do this at top-level! `dml.globals.dml_version` can't be well-defined here, can it? Even if it is, that feels like that would _very_ easily break from any change we do re. when dml version is determined.

Reply:
Oh, crap. Will fix it somehow.